frontend/src/lib/colors.ts

```diff
@@ -82,7 +82,7 @@ export const dashboardColorHSL = {
     blue: [212, 63, 40],
     purple: [249, 46, 51],
     green: [145, 60, 34],
-    black: [0, 0, 0],
+    black: [0, 0, 18],
```

Comment:
Too dark? :D

Reply:
Hahaha [this](https://news.mit.edu/2019/blackest-black-material-cnt-0913) is too dark. Just reflecting the new designs, do think it looks a lot better. Also remember [reading this](https://blog.superhuman.com/how-to-design-delightful-dark-themes/) where it suggest never to use pure white/black to avoid eye strain.